phys2bids/physio_obj.py

```diff
@@ -504,7 +512,7 @@ def check_trigger_amount(self, thr=None, num_timepoints_expected=0, tr=0):
                         'were not provided.')
         self.thr = thr
         self.time_offset = time_offset
-        self.timeseries[0] -= time_offset
+        self.timeseries[0] = self.timeseries[0] - time_offset
```

Comment:
Why have this change?

Reply:
It's [this commit](https://github.com/physiopy/phys2bids/pull/308/commits/2d3e378b7f1ecf18b0ea482008ad3b9837bcc924). Basically, while testing I got some dtype errors of impossible subtraction between dtype X and dtype Y with some peculiar cases of `self.timeseries[0]` and `time_offset`. I read that an easy way to avoid them is using a complete subtraction statement rather than the previous version. It's a more global solution (despite possibly less elegant).

Reply:
That makes sense. I've had issues with in-place subtraction in the past too. Thanks for the explanation!